GSA-TTS/FAC
# OMB Control # and Expiration Date to Header

# OMB Control Number and Expiration Date to the Header

Issue: https://github.com/GSA-TTS/FAC/issues/1900

## Changes:
1. OMB Control number and expiration date are strings in settings. They are added to the template context processors by way of `/config/context_processors.py`, making them global. They are accessed in the header on every page.
2. This affects no tests.

## How to Test:
Switch to this branch and run normally. Verify that the OMB control number and expiration date appear on any page with the "blue bar" under the header. That's every page that isn't the "sign-in" homepage. 

## Screenshot:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/fa512f53-a0e5-4a0a-a4ac-a8f45b9a30bd)

## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
